kubectl-fdb/cmd/root.go

```diff
@@ -67,13 +72,20 @@ func NewRootCmd(streams genericclioptions.IOStreams) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return cmd.Help()
 		},
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			versionCheck, _ := cmd.Flags().GetBool("version-check")
+			if versionCheck {
+				skipCommand = !usingLatestPluginVersion(cmd)
+			}
+		},
 	}
 	cmd.SetOut(o.Out)
 	cmd.SetErr(o.ErrOut)
 	cmd.SetIn(o.In)
 
 	viper.SetDefault("license", "apache 2")
 	cmd.PersistentFlags().StringP("operator-name", "o", "fdb-kubernetes-operator-controller-manager", "Name of the Deployment for the operator.")
+	cmd.PersistentFlags().BoolP("version-check", "v", true, "If the plugin should compare its version with latest release to see if it is the latest.")
```

Comment:
```suggestion
	cmd.PersistentFlags().Bool("version-check", true, "If the plugin should check if a newer release of the plugin exists. If so the command will not be executed.")
```

We can drop the shorthand version of this flag.